mem-05-H.xml

```diff
@@ -7763,14 +7816,14 @@ This verb is also used for regular polygons and polyhedra.[2]</column>
       <column name="definition_de">Galle, Schleim</column>
       <column name="definition_fa">لجن [AUTOTRANSLATED]</column>
       <column name="definition_sv">galla, slem</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">тина, слизь, муть</column>
       <column name="definition_zh_HK">粘液 [AUTOTRANSLATED]</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{char:v}, {quy'Ip:n}, {chej:n}</column>
-      <column name="notes">Depending on the context, this word may also be glossed as "bile" or "gall".
+      <column name="notes">В зависимости от контекста, это слово может означать "желчь".
```

Comment:
Didn't catch this at the time, but someone just pointed out to me that the Russian has replaced the English notes here.